pages/careers/_open-positions.html

```diff
@@ -237,5 +320,267 @@ <h4>Help us a create a fundamentally better DevOps experience!</h4>
+        methodologies that will help us translate our shared vision into a fantastic experience for customers.
+      </li>
+      <li>
+        <strong>Leadership.</strong>
+        We're looking for someone with a proven track record of delivering products that move the needle. You must
+        have the ability to lead a team through ideation, prototyping, user testing, development, launch, and iteration.
+      </li>
+      <li>
+        <strong>Domain expertise.</strong>
+        We don't expect you to know everything coming in, but to create products in our space, you must be able to
+        quickly build a deep understanding of our domain: DevOps, cloud software, software delivery, CI / CD, and
+        infrastructure as code. If you happen to have prior experience in software engineering, working on dev tools, o
+        other technical domains, that is a big plus.
+      </li>
+      <li>
+        <strong>Values.</strong>
+        You are inspired by <a href="/about/#our-values">our values</a>. Bonus points for a sense of humor, empathy,
+        and curiosity.
+      </li>
+    </ul>
+    <h4>Compatible time zones</h4>
+    <p>
+      You'll be working with a team in the <strong>US time zones</strong>, so you can be located in almost any
+      country as long as your time zone is no further west than Los Angeles (GMT-8) and no further east than New York
+      (GMT-4). We've found that when everyone on the team is located in similar time zones, it's easier to collaborate
+      and there's much less pressure to stay up late or get up early, so this is a hard constraint, even if you're
+      willing to work hours different from your current time zone.
+    </p>
+    <h4>Help us a create a fundamentally better DevOps experience!</h4>
+    <p>
+      If you're excited by the above:<br />
+      <!-- TODO: Fill in the proper link!!! -->
+      <a
+        href="#TODO:fill-me-in!"
+        class="btn btn-primary-hollow btn-sm d-inline-block btn-dpa"
+        style="margin-top: 10px"
+        role="button"
+      >Apply Now</a
+      >
+    </p>
+
+    <hr />
+
+    <h3 id="product-manager-brand-new-product" style="margin-top: 5px; margin-bottom: 5px">Senior Product Manager - Brand New Product</h3>
+    <div style="background-color:#2e3b4a; padding:4px 6px; border-radius: 10px; display:inline-block; font-size:14px;"><strong>Time Zone:</strong> European time zones</div>
+
+    <h4>What You'll Work On</h4>
+    <ul>
+      <li>
+        <strong>Create a brand new product.</strong>
+        Take ownership of a new, unannounced product that we have in the works, which includes both a beautiful,
+        first-class CLI experience and a powerful web UI. Join a small team to help design, prototype, test, launch,
+        and develop this exciting new initiative that we believe will transform the software industry.
+      </li>
```

Comment:
Everyone loves to build a brand new product so I worry this line will succeed in rousing people's interest but also flood us with poor fits. I'd prefer to mitigate that by being more specific like a "new product in DevOps developer experience," or something similar.

Reply:
I'd rather bias us towards getting more super excited candidates, even if it means having to filter out a few extra poor fits. I'm looking for standout candidates, so boosting variance is more valuable than a higher mean.